batch/Dockerfile.worker

```diff
@@ -34,6 +34,7 @@ RUN python3 -m pip install --upgrade --no-cache-dir pip
 
 COPY hail/python/setup-hailtop.py /hailtop/setup.py
 COPY hail/python/hailtop /hailtop/hailtop/
+COPY /hail_version /hailtop/hailtop/hail_version
```

Comment:
Missed this earlier. Don't you need to copy the MANIFEST.in here?

Reply:
Yes, thank you! I've addressed both these in f6d3580.